OpenRA.GameMonitor/GameMonitor.cs

```diff
@@ -15,16 +15,44 @@
 using System.Media;
 using System.Reflection;
 using System.Windows.Forms;
+using Microsoft.Win32;
 
 namespace OpenRA
 {
 	class GameMonitor
 	{
+		static readonly string[] VisualStudioRuntimeVersions = { "10.0", "11.0", "12.0" };
+
 		static Process gameProcess;
 
 		[STAThread]
 		static void Main(string[] args)
 		{
+			if (Platform.CurrentPlatform == PlatformType.Windows)
+			{
+				var visualStudioRuntimeInstalled = 0;
+
+				foreach (var version in VisualStudioRuntimeVersions)
+				{
+					var key = Registry.LocalMachine.OpenSubKey(@"SOFTWARE\Microsoft\DevDiv\VC\Servicing\{0}\RuntimeMinimum\".F(version), false);
+					if (key != null && visualStudioRuntimeInstalled == 0)
+						visualStudioRuntimeInstalled = (int)key.GetValue("Install", 0);
+
+					key.Close();
```

Comment:
If you fail to open (find) the key, you get an NRE here. You do have a null check above though.